egov-persister/attendance-service-persister.yml

```
@@ -0,0 +1,410 @@
+   name: Attendance Log
+   description: Persists attendance logs and corresponding documents
+   fromTopic: save-attendance-log-health
+   isTransaction: true
+   isAuditEnabled: true
+   module: ATND
+   objecIdJsonPath: $.id
+   tenantIdJsonPath: $.tenantId
+   transactionCodeJsonPath: $.registerId
+   auditAttributeBasePath:  $.attendance.*
+   queryMaps:
+    - query: INSERT INTO eg_wms_attendance_log(id,clientreferenceid,tenantid,individual_id,register_id,status,time,event_type,additionaldetails,createdby,lastmodifiedby,createdtime,lastmodifiedtime, clientcreatedby, clientlastmodifiedby, clientcreatedtime, clientlastmodifiedtime) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?);
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 3 but found 4

(indentation)